app/src/main/java/com/vuongnq14798/vuongmp3/data/source/TracksRepository.kt

```diff
@@ -0,0 +1,24 @@
+package com.vuongnq14798.vuongmp3.data.source
+
+import com.vuongnq14798.vuongmp3.data.source.local.TracksLocalDataSource
+import com.vuongnq14798.vuongmp3.data.source.remote.TracksRemoteDataSource
+
+class TracksRepository(val mTracksRemoteDataSource: TracksRemoteDataSource,
```

Comment:
Với những class có header dài thì mình trình bày xuống dòng theo conventiosn sau đây nhé: https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting
```suggestion
class TracksRepository(
    val mTracksRemoteDataSource: TracksRemoteDataSource,
    val mTracksLocalDataSource: TracksLocalDataSource
): TracksDataSource.Remote, TracksDataSource.Local {
```
theo ông thì tại sao ở đây lại để là val mà không phải là private val ?